packages/rum-core/src/domain/tracing/tracer.ts

```diff
@@ -113,11 +115,11 @@ function getCrypto() {
   return window.crypto || (window as any).msCrypto
 }
 
-function makeTracingHeaders(traceId: TraceIdentifier, spanId: TraceIdentifier): TracingHeaders {
+function makeTracingHeaders(traceId: TraceIdentifier, spanId: TraceIdentifier, traceSampled: boolean): TracingHeaders {
   return {
     'x-datadog-origin': 'rum',
     'x-datadog-parent-id': spanId.toDecimalString(),
-    'x-datadog-sampling-priority': '1',
+    'x-datadog-sampling-priority': traceSampled ? '1' : '0',
     'x-datadog-trace-id': traceId.toDecimalString(),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏FMU, if `x-datadog-sampling-priority` is `0`, the agent won't create a trace. So do we need the `x-datadog-parent-id` and `x-datadog-trace-id` in this case?

Reply:
> FMU, if x-datadog-sampling-priority is 0, the agent won't create a trace.

FMU, the backend could still create the trace if it does not have enough but we will not have a root span in this case. We could not have the tracing headers but the idea was to have an implementation more close to the next step.
Don't hesitate to have a second look at Xavier's RFC.

Reply:
Ok I've taken another look and it makes sense. Maybe add a comment to give a bit of context?

Reply:
sure